kea-vscode/src/views/common/base-files-root-tree-node.ts

```diff
@@ -37,20 +39,20 @@ export abstract class BaseFilesRootTreeNode implements IParentTreeNode<FilesRoot
       const folderName = pathParts[idx];
       const folderPath = pathParts.slice(0, idx + 1).join("/");
 
-      let folderNode = parents.find((node) => node instanceof FolderTreeNode && node.folderName === folderName);
+      let folderNode = parents.find((node) => node instanceof RemoteFolderTreeNode && node.folderName === folderName);
       if (folderNode === undefined) {
-        folderNode = new FolderTreeNode(folderPath);
+        folderNode = new RemoteFolderTreeNode(folderPath);
         parents.push(folderNode);
       }
 
-      parents = (folderNode as FolderTreeNode).children;
+      parents = (folderNode as RemoteFolderTreeNode).children;
     }
 
     const comments = reviewComments.filter((comment) => comment.path === file.filename);
     const fileName = pathParts[pathParts.length - 1];
-    const fileNode = new FileTreeNode(this._repository.account.accountKey, this._repository.repoId, file, comments);
+    const fileNode = new RemoteFileTreeNode(this._repository.account.accountKey, this._repository.repoId, file, comments);
```

Comment:
I can't see why remote files/folders would be used here. Try use local ones where possible.